server/homebrew.api.js

```diff
@@ -308,7 +311,7 @@ If you believe you should have access to this brew, ask the file owner to invite
 				await HomebrewModel.deleteOne({ _id: brew._id })
 					.catch((err)=>{
 						console.error(err);
-						throw { status: 500, message: 'Error while removing' };
+						throw { ...new Error, status: 500, message: 'Error while removing' };
```

Comment:
I *think* this is equivalent to:

```suggestion
						throw { Error('Error while removing', {status: 500}) };
```

By default the "Error" object uses its first field as the `message:`. This reads more clearly to me without needing the spread operator, etc.

Reply:
At this point, I've shifted to using `throw { [...] }`, which allows us to add custom fields. We can revisit this if necessary in the current commit.